e2e-test/business/factories/MultiSigDAOFactory.ts

```diff
@@ -24,7 +24,7 @@ const UPGRADE_SAFE_FACTORY_LOGIC_IMPL = "upgradeSafeFactoryAddress";
 
 export default class MultiSigDAOFactory extends Base {
   initialize = async (
-    daoConfigDetails: DAOConfigDetails = DEFAULT_DAO_CONFIG,
+    daoConfigDetails: FeeConfigDetails = DEFAULT_FEE_CONFIG,
```

Comment:
```suggestion
    daoConfigDetails: FeeConfigDetails = DEFAULT_FEE_CONFIG,
```Can we rename it too something feeConfigDetails, also their usage ?